### PR TITLE
Add a profile for myself (Stephen Swatman)

### PR DESCRIPTION
**Pull request by stephenswat**
I'm responding to the mail sent out to the training mailing list requesting everyone to update their profiles. Since I don't have a profile yet, I think it would be wise to make one!

### CODE REVIEW

**Review by klieret — APPROVED** [review]
Thanks a lot for adding yourself!

File: _profiles/stephen_swatman.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+# Required:
+title: Stephen Nicholas Swatman
+country: FR
+
+# Optional (please add github or gravatar for your picture)
+github: stephenswat
+gravatar: null
+homepage: http://v25.nl/
```
Comment (klieret) [inline _profiles/stephen_swatman.md:9]: hmm, I get an E 503 for this

**Review by klieret — CHANGES_REQUESTED** [review]

File: _profiles/stephen_swatman.md (hunk 1 above)
Comment (stephenswat) [reply _profiles/stephen_swatman.md:9]: My mistake, was a bad configuration. I've had that site under "temporary" maintenance for a while now, but I promise I'll put something there soon. :sweat_smile:

**Review by klieret — APPROVED** [review]

**Comment by klieret** [comment]
Thanks again!

**Pull request merged**